### DISCUSSION TITLE
Open sequelace as quick action

### DISCUSSION THREAD

**Discussion by sebastian-ehrling** [php-perfect/ddev-intellij-plugin#226] [Ideas]
Is it possible to add an action "Open DB in sequelace" to the action bar at the bottom?

**Comment by rfay** [comment]
You know that the plugin already provides direct integration with PhpStorm's database browser right?

**Comment by sebastian-ehrling** [comment]
Yes, of course! But my favourite database tool is sequelace and not the integreated database browser.

**Reply by rfay** [reply]
I guess the question is... should everybody's favorite be supported? There are like a dozen of them, including TablePlus and adminer, etc.

**Comment by nico-loeber** [comment]
The action bar at the bottom shows the status of the ddev related containers and allows you to quick access services provided by them. This means URLs that are exposed to ddev, like the url of the website, mailhog and in the past phpmyadmin as it was an exposed service of the container.
If you add a management service like kibana as an extra service it should appear automatically in this bar.

As far as I know sequelace is not such a service and so it won't appear in the container status bar.

So the answer is no I'm afraid.
